src/pages/Dashboard.tsx

```diff
@@ -66,6 +77,14 @@ const Dashboard = () => {
 
   return (
     <div className="grid grid-cols-7 gap-6">
+      {/* Banner on top showing optional information about membership status */}
+      {/* TODO: Hidde will make a nice banner styling */}
```

Comment:
Make a story in Jira instead of TODO comments (strictly disallowed by our chairman)